src/power_grid_model/validation/errors.py

```diff
@@ -49,6 +49,8 @@ class ValidationError(ABC):
 
     _message: str = "An unknown validation error occurred."
 
+    delimiter: str = " and "
```

Comment:
Nitpick: make private

```suggestion
    _delimiter: str = " and "
```

Reply:
Thank you for the nice feedback! Sure, you can announce this in your next meetup!
I have made the requested change and made the delimiter private.